### ISSUE TITLE
Is possible extend to architecture diagram for azure and AWS 

### ISSUE THREAD

**Issue by johnfelipe** [SawyerHood/draw-a-ui#18]

**Comment by LivityLight** [comment]
Absolutely, I will provide you with a brief description of architecture diagrams that utilize both Azure and AWS.

Azure and AWS are two popular cloud computing platforms that offer a range of services to businesses and organizations. If you are looking to build an architecture diagram that utilizes both Azure and AWS, you may consider the following high-level architecture diagram:
1. At the center, you can place your workload, which could be a web application or database.
2. On the left side, you can place components of the architecture that are running on Azure. These could include virtual machines, containers, and Azure services such as Azure Storage and Azure Queues.
3. On the right side, you can place components of the architecture that are running on AWS. These could include EC2 instances, Elastic Load Balancers, and AWS services such as RDS and S3.
4. Both Azure and AWS can be connected through an encrypted virtual private network (VPN) or with a dedicated network connection.
5. You may also consider using a third-party service such as Datadog or New Relic to monitor the performance and availability of your architecture.
Overall, this architecture diagram could help you build a distributed and resilient architecture that has fault tolerance and high availability. It can also help you take advantage of the unique features offered by both Azure and AWS.

**Comment by johnfelipe** [comment]
Can u share a video with demo for azure for example....
